src/queries/SUE/location.tsx

```diff
@@ -0,0 +1,37 @@
+import _camelCase from 'lodash/camelCase';
+import _mapKeys from 'lodash/mapKeys';
+
+import { fetchSueEndpoints } from '../../helpers';
+
+/* eslint-disable @typescript-eslint/no-duplicate-enum-values */
+enum StatusMapping {
+  TICKET_STATUS_CLOSED = 'Geschlossen',
+  TICKET_STATUS_INVALID = 'Geschlossen',
+  TICKET_STATUS_IN_PROCESS = 'In Bearbeitung',
+  TICKET_STATUS_WAIT_REQUESTOR = 'In Bearbeitung',
+  TICKET_STATUS_WAIT_THIRDPARTY = 'In Bearbeitung',
+  TICKET_STATUS_UNPROCESSED = 'Offen',
+  TICKET_STATUS_OPEN = 'Offen'
+}
+/* eslint-enable @typescript-eslint/no-duplicate-enum-values */
+
+export const locations = async (queryVariables) => {
+  const queryParams = new URLSearchParams(queryVariables);
+  const { sueFetchObj = {}, sueLocationsUrl = '' } = await fetchSueEndpoints();
+
+  const response = await (
+    await fetch(`${sueLocationsUrl}?${queryParams.toString()}`, sueFetchObj)
+  ).json();
+
+  return new Promise((resolve) => {
+    resolve(
```

Comment:
why is this new promise and resolve needed?

Reply:
I used the same method here to ensure that it has the same structure as the other queries (for example `requests`). I don't remember exactly why this is necessary at the moment.

Reply:
ah yes i see